docker/ci/scripts/prepare_unit_tests.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# Script for preparing unit tests within a docker container. Used by
+# unit_tests.sh.
+
+source docker/ci/scripts/prepare_tests_common.sh
+
+export CI_JOB=unit_tests
+
+# Update the locals.yml which was initialized by prepare_tests_common.sh with
+# job-specific values
+echo "
+# Unit test settings and secrets
+google_maps_api_key: boguskey
```

Comment:
this can be removed: https://github.com/code-dot-org/code-dot-org/blob/b21520909b80cf793af2f2cc20916b7e6f729964/config/test.yml.erb#L26